### PR TITLE
[2148] General snagging - confirm page

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/ktwCe1bJ/2148-general-snagging-confirm-page

### Changes proposed in this pull request

* All confirm page titles start with "Confirm trainee's xxx"

### Guidance to review
* Complete each section to see confirm page

### CODE REVIEW

**Comment by mooktakim** [comment]
[General snagging - confirm page](https://trello.com/c/ktwCe1bJ/2148-general-snagging-confirm-page)

**Review by AbigailMcP — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ en:
         blank: "Draft record"
     confirmation:
       not_provided: Not provided
-      heading: Confirm %{section_title}
+      heading: "Confirm trainee's %{section_title}"
```
Comment (defong) [inline config/locales/en.yml:38]:
```suggestion
      heading: "Confirm trainee’s %{section_title}"
```

File: spec/features/form_sections/personal_and_education_details/degrees/delete_degree_spec.rb [hunk 2]
```diff
@@ -36,7 +36,7 @@ def and_i_click_the_delete_button
   end
 
   def then_i_should_not_see_any_degree
-    expect(degrees_confirm_page.page_heading.text).to eql("Confirm degree details")
+    expect(degrees_confirm_page.page_heading.text).to eql("Confirm trainee's degree details")
```
Comment (defong) [inline spec/features/form_sections/personal_and_education_details/degrees/delete_degree_spec.rb:39]:
```suggestion
    expect(degrees_confirm_page.page_heading.text).to eql("Confirm trainee’s degree details")
```

**Review by defong — APPROVED** [review]
Smart quotes?

¯\_(ツ)_/¯

**Pull request merged**